keyboards/wheatfield/fanoe_hotswap/keymaps/default/keymap.c

```diff
@@ -0,0 +1,55 @@
+#include QMK_KEYBOARD_H
+#include <print.h>
+
+#define _BL 0
+#define _FL 1
```

Comment:
Looks like these two symbols are unused. They can be removed.

Reply:
OK removed them